config/services.yml

```diff
@@ -14,44 +14,45 @@ services:
   prestashop.module.keycloak_connector_demo.client:
     class: GuzzleHttp\Client
 
-  PrestaShop\KeycloakConnectorDemo\RequestBuilder:
-    class: PrestaShop\KeycloakConnectorDemo\RequestBuilder
+  PrestaShop\Module\KeycloakConnectorDemo\RequestBuilder:
+    class: PrestaShop\Module\KeycloakConnectorDemo\RequestBuilder
     arguments:
       - '@prestashop.adapter.legacy.configuration'
       - '@prestashop.module.keycloak_connector_demo.php_encrypt'
 
-  PrestaShop\KeycloakConnectorDemo\Form\ConfigurationDataConfiguration:
-    class: PrestaShop\KeycloakConnectorDemo\Form\ConfigurationDataConfiguration
+  PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationDataConfiguration:
+    class: PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationDataConfiguration
     arguments:
       - '@prestashop.adapter.legacy.configuration'
       - '@prestashop.module.keycloak_connector_demo.php_encrypt'
-      - '@PrestaShop\KeycloakConnectorDemo\RequestBuilder'
+      - '@PrestaShop\Module\KeycloakConnectorDemo\RequestBuilder'
       - '@prestashop.module.keycloak_connector_demo.client'
 
-  PrestaShop\KeycloakConnectorDemo\Form\ConfigurationDataProvider:
-    class: PrestaShop\KeycloakConnectorDemo\Form\ConfigurationDataProvider
+  PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationDataProvider:
+    class: PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationDataProvider
     arguments:
-      - '@PrestaShop\KeycloakConnectorDemo\Form\ConfigurationDataConfiguration'
+      - '@PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationDataConfiguration'
 
   prestashop.module.keycloak_connector_demo.form.configuration_data_handler:
     class: 'PrestaShop\PrestaShop\Core\Form\Handler'
     public: true
     arguments:
       - '@form.factory'
       - '@prestashop.core.hook.dispatcher'
-      - '@PrestaShop\KeycloakConnectorDemo\Form\ConfigurationDataProvider'
-      - 'PrestaShop\KeycloakConnectorDemo\Form\ConfigurationType'
+      - '@PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationDataProvider'
+      - 'PrestaShop\Module\KeycloakConnectorDemo\Form\ConfigurationType'
       - 'KeycloakConfiguration'
 
-  PrestaShop\KeycloakConnectorDemo\Controller\ConfigurationController:
-    class: PrestaShop\KeycloakConnectorDemo\Controller\ConfigurationController
+  PrestaShop\Module\KeycloakConnectorDemo\Controller\ConfigurationController:
+    public: true
+    autowire: true
```

Comment:
This right here is the compatibility for PS9, everything else is namespace fix

Reply:
Would this mean that _every module developer has to put `autowire: true`  on his symfony module controllers_, else they don't work?

If yes
- we need to understand why (to make sure this is not a symptom of a bigger problem)
- we need to document it